capabilities/version-control.md

```diff
@@ -1,28 +1,109 @@
 # [Version Control](https://dora.dev/devops-capabilities/technical/version-control/)
 
-Version control allows teams to revert changes back to a working state. When this is fast an easy to accomplish, teams can move forward with less fear of breaking the system. Technologies like Git and SVN have made version control for code popular. However, this DORA Capability applies all the same to configuration, data, infrastructure, and documentation. Furthermore, version control is foundational for enabling automation, ensuring disaster recovery, and facilitating continuous integration.
+Comprehensive version control ensures reproducibility and traceability across the system, enabling teams to provision production-like environments, track changes, and respond quickly to evolving needs. This capability enhances disaster recovery, auditability, and quality by allowing teams to restore services efficiently, demonstrate process integrity, and receive rapid feedback on changes. When teams can scale their environments and roll back to verified working states, when necessary, they can support capacity management and defect response.
+
+Without a robust version control strategy, teams are left vulnerable to disruptions in productivity and system reliability. They may struggle to reproduce environments, leading to prolonged downtime and inefficiency. Auditability is also compromised, making regulatory compliance difficult to demonstrate. Defect response suffers, as teams lack the ability to swiftly restore a known working state. 
+
+By prioritizing effective version control practices, organizations can mitigate these risks and ensure seamless, reliable, and compliant software delivery.
+
+## Nuances
+
+This section outlines common pitfalls, challenges, or limitations teams commonly encounter when applying this capability. The goal here is not to discourage you. Rather, the goal is to arm you with the appropriate context so that you can make an informed decision about when and how to implement the capability with your teams.
+
+### Deterministic Builds vs. Up-to-Date Dependencies
+
+Version control systems often face a trade-off between deterministic builds and up-to-date dependencies. Some developers prefer specifying dependencies to use the latest versions at build time for enhanced security. However, this compromises build determinism, making it challenging to reproduce identical builds. Tools like [Dependabot](https://github.com/dependabot) can mitigate this issue by automatically creating pull requests to update dependencies, separating dependency updates from the build process.
```

Comment:
Is your target reader familiar with the term "deterministic builds"? If some may not be, then we could add a brief definition between the first and second sentences here.

Reply:
Yeah, I think that's a safe one to assume. We can favor brevity here.